osmo-test-5/osmosis.zone_assets.json

```diff
@@ -104,6 +104,12 @@
       "base_denom": "uixo",
       "path": "transfer/channel-1637/uixo",
       "osmosis_verified": true
+    },
+    {
+      "chain_name": "archwaytestnet",
+      "base_denom": "aconst",
+      "path": "transfer/channel-7779/aconst",
```

Comment:
This doesn't look to be the correct channel. The registered connection specifies channel 3938
https://github.com/cosmos/chain-registry/blob/a0f3cab5543e2cd8070c7030374c18de570e3a27/testnets/_IBC/archwaytestnet-osmosistestnet.json#L20

Reply:
> Hi @JeremyParish69 I made another PR to Cosmos chain registry here: https://github.com/cosmos/chain-registry/pull/4924
> 
> What I put there is the active channel now from Archway

Yeah I think the other one is very old and abandoned. Does it work to merge this alongside one to change the ibc registry entry for that testnet?

Reply:
Hi @Cordtus I think it is fine to merge this as we got the channel id also from Archway team and the PR on cosmos chainregistry was merged already

Reply:
Hi @JeremyParish69 can u help to rerun the validation ? Thanks